InventoryService/cmd/controllers/item_controller.go

```diff
@@ -0,0 +1,205 @@
+package controllers
+
+import (
+	"encoding/json"
+	"inventoryService/m/v2/cmd/models"
+	"inventoryService/m/v2/cmd/resources"
+	"inventoryService/m/v2/ent"
+	"inventoryService/m/v2/ent/ingridient"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type ItemController struct {
+	Client *ent.Client
+}
+
+type Response struct {
+	Msg  string      `json:"message"`
+	Data interface{} `json:"data"`
+}
+
+func ResponseJSON(c *gin.Context, httpCode int, msg string, data interface{}) {
```

Comment:
Anyway it's better to have `Response` and `ResponseJSON` in separate file because it will be common for all controllers